django_rq/views.py

```diff
@@ -144,6 +149,43 @@ def failed_jobs(request, queue_index):
     return render(request, 'django_rq/jobs.html', context_data)
 
 
+@staff_member_required
+def scheduled_jobs(request, queue_index):
+    queue_index = int(queue_index)
+    queue = get_queue_by_index(queue_index)
+
+    registry = ScheduledJobRegistry(queue.name, queue.connection)
+
+    items_per_page = 100
+    num_jobs = len(registry)
+    page = int(request.GET.get('page', 1))
+    jobs = []
+
+    if num_jobs > 0:
+        last_page = int(ceil(num_jobs / items_per_page))
+        page_range = range(1, last_page + 1)
+        offset = items_per_page * (page - 1)
+        job_ids = registry.get_job_ids(offset, offset + items_per_page - 1)
+
+        jobs = Job.fetch_many(job_ids, connection=queue.connection)
```

Comment:
If one of the jobs is no longer in Redis, `None` will be returned. In this case we should remove the job from registry. Will need a test case for this.

Reply:
I am not so sure, that the view should clear the registry, but ok. And I guess, that this logic should be shared by other views at some point.